### ISSUE TITLE
Panic when rapidly changing visibility options in live editor

### ISSUE THREAD

**Issue by mitchgrout** [slint-ui/slint#5697]
Starting with an empty component, open the live editor, and using tab, select the check box for the `visibility` attribute. Rapidly hit the space bar to quickly toggle the state of the attribute. This triggers the following panic:

```
thread 'main' panicked at D:\a\slint\slint\target\x86_64-pc-windows-msvc\release\build\slint-lsp-8eb16ecdb39ea5da\out\main.rs:27116:167:
called `Option::unwrap()` on a `None` value
```

Triggered on VSCode Slint v1.7.1 and SlintPad.
Surprisingly this cannot be triggered by rapidly _clicking_ the check box.

**Comment by tronical** [comment]
This seems the offending generated code in question:

```
 let cache = (InnerComponent_rectangle_30 :: FIELD_OFFSETS . r#rectangle_30_empty_31_layout_cache) . apply_pin (_self . parent . upgrade () . unwrap () . as_pin_ref ()) . get () ;
```

**Comment by ogoffart** [comment]
I couldn't reproduce the problem.
But I have the suspicion that this is related to https://github.com/slint-ui/slint/issues/3464

**Comment by ogoffart** [comment]
https://github.com/slint-ui/slint/pull/5813 might have fixed the problem.

**Comment by tronical** [comment]
We can't reproduce this. We'd be very grateful if you could try again with the 1.7.2 release (that we're rolling today).

**Comment by tronical** [comment]
Thanks for checking ❤️
